src/app/home/panel/cards/transactions/filter-form/filter-form.component.ts

```diff
@@ -32,10 +35,17 @@ export class FilterFormComponent implements OnInit {
   }
 
   saveFilter(form) {
-    //TODO
+    this.transactionsListFiltersDto.date = form.date;
+    this.transactionsListFiltersDto.amountFrom = form.from;
+    this.transactionsListFiltersDto.amountTo = form.to;
+    this.transactionsListFiltersDto.type = this.selectedType;
+    this.transactionsListFiltersDto.budgets = this.budgetsList.filter(element => element.selected).map(budget => budget.id);
+
+    this.panelService.sendTransactionsListFilters(this.transactionsListFiltersDto)
```

Comment:
There is no need of using class field if it's only being used in a single method. Just use it locally.